Cargo.toml

```
@@ -44,6 +44,10 @@ walkdir = "2.5.0"
 wdl = { version = "0.15.1", features = ["cli", "codespan", "engine", "lsp", "doc"] }
 
 
+[patch.crates-io]
+wdl = { git = "https://github.com/stjude-rust-labs/wdl.git", branch = "main" }
+
+
```

I just took the current head of the `wdl` repo for the revision here, so it might be worth double checking this is what is expected. `patch` sections are best used for temporary overrides in local development, and a symbolic branch name that changes over time is not good for CI stability.

```suggestion
wdl = { version = "0.15.1", features = ["cli", "codespan", "engine", "lsp", "doc"], git = "https://github.com/stjude-rust-labs/wdl.git", rev = "f7ba4c0d16a6b16c7b3bf7d302658799a0eed77f" }
```

The better answer for this is bringing in `wdl` as a submodule or combining the repos, which I am going to write up into a full proposal soon.